af/fastpath/fastpath/core.py

```diff
@@ -1679,35 +1696,55 @@ def core():
         clean_caches()
 
 
-def setup_fingerprints():
-    """Setup fingerprints lookup dictionary
-    "ZZ" applies a fingerprint globally
-    """
-    # pre-process fingerprints to speed up lookup
+def extract_expected_countries(ec):
```

Comment:
Is this function still being used? I see it only mentioned in the unittests.

Reply:
I'd rather keep it around to make the code testable in case we add more clever logic in future.
Refactored to use it. (FWIW)